README.md

```diff
@@ -549,6 +550,36 @@ ln -s /usr/share/zsh/plugins/powerlevel10k ~/.local/share/zsh/plugins/
 Follow the instructions on
 [this page](https://fig.io/plugins/other/powerlevel10k).
 
+### NixOS
+
+**Home Manager**
+```nix
+programs.zsh = {
+  ...
+  plugins = [
+      {
+        name = "zsh-powerlevel10k";
+        src = "${pkgs.zsh-powerlevel10k}/share/zsh-powerlevel10k/";
+        file = "powerlevel10k.zsh-theme";
+      }
+    ];
+
+    initExtra = ''
```

Comment:
`initExtra` will be deprecating in the next release (see [here](https://github.com/nix-community/home-manager/blob/master/modules/programs/zsh.nix)), maybe we want to specify it is for 24.11 or earlier

for 25.05 or later use this
```nix
initContent = let
	initExtra = lib.mkOrder 1000 ''
		[[ ! -f ~/.p10k.zsh ]] || source ~/.p10k.zsh
	'';
in
lib.mkMerge [nitExtra];
```